Cargo.toml

```diff
@@ -225,6 +225,9 @@ ed25519-dalek = { git = "https://github.com/mobilecoinfoundation/ed25519-dalek.g
 mbedtls = { git = "https://github.com/mobilecoinfoundation/rust-mbedtls.git", rev = "ac6ee17a31e37311ce7f4fa0649c340e5d85258d" }
 mbedtls-sys-auto = { git = "https://github.com/mobilecoinfoundation/rust-mbedtls.git", rev = "ac6ee17a31e37311ce7f4fa0649c340e5d85258d" }
 
+# mc-oblivious-aes-gcm patched to 0.10.1
+mc-oblivious-aes-gcm = { git = "https://github.com/mobilecoinfoundation/oblivious-aes-gcm.git", rev = "0855242eaee83ca9eb01d729482f8c029a49ef0b" }
+
```

Comment:
We'll need to wait on https://github.com/mobilecoinfoundation/oblivious-aes-gcm/pull/5 to get merged prior to merging this change, and we may want to bump this sha to the merged one.

Reply:
updated to main for https://github.com/mobilecoinfoundation/mobilecoin/pull/2806/commits/fce976247964fec94f0b012d2c54b1afebc61a20